doc/release/RELEASE-NOTES.md

```
@@ -9,8 +9,57 @@ This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html
 * Product name: Open Supply Hub
 * Release date: Jun 28, 2025
 
+### Database changes
+* *Describe high-level database changes.*
+
+#### Migrations:
```

markdownlint-cli2 (0.17.2)

Trailing punctuation in heading
Punctuation: ':'

(MD026, no-trailing-punctuation)